src/Conference/ConferenceDetails.jsx

```diff
@@ -73,16 +75,30 @@ const ConferenceDetails = () => {
   const { name, topic, dateTime, city, description } = conference || {};
   const date = new Date(dateTime);
 
+  const deleteConference = async id => {
+    try {
+      await deleteConferenceById(id);
+      history.push("/");
```

Comment:
You want to put the toast emitter in here, and maybe another one in the catch for the error :)

Reply:
Actually just realised you did it in the api file, up to you :) But you didn't need the pauseOnHover bits in that file btw